docs/developer-docs/security/security-best-practices-references.mdx

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 5
+sidebar_label: Further Reading
+keywords: [security, concept]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Security Best Practices: Further Reading and Resources
+
+<MarkdownChipRow labels={["Security", "Concept"]} />
+
+These are some resources for further reading. They can be useful for developing secure dapps or security reviewing them.
```

Comment:
I'd like to highlight that this is not just a "further reading" section which means it's less important than the IC specific best practices. Almost nobody does "further reading" and might prioritize our own best practices above these technology specific guides.

```suggestion
## Overview 
	Below are resources which cover security best practices for technologies you are likely using in your dapp. These best practices are equally as important as our Internet Computer specific guidelines and should be studied carefully. They can be useful to reference when developing secure dapps or executing security reviews.
```